modules/bidResponseFilter/index.js

```diff
@@ -0,0 +1,39 @@
+import { config } from '../../src/config.js';
+import { getHook } from '../../src/hook.js';
+
+export const MODULE_NAME = 'bidResponseFilter';
+export const PUBLISHER_FILTER_REJECTION_REASON = 'PUBLISHER_FILTER_REJECTION_REASON';
+export const BID_CATEGORY_REJECTION_REASON = 'BID_CATEGORY_REJECTION_REASON';
+export const BID_ADV_DOMAINS_REJECTION_REASON = 'BID_ADV_DOMAINS_REJECTION_REASON';
+export const BID_ATTR_REJECTION_REASON = 'BID_ATTR_REJECTION_REASON';
+
+function init() {
+  getHook('addBidResponse').before(addBidResponseHook);
+};
+
+export function addBidResponseHook(next, adUnitCode, bid, reject) {
+  const { bcat = [], badv = [], battr = [] } = config.getAnyConfig('ortb2') || {};
+  const moduleConfig = config.getConfig(MODULE_NAME);
+  
+  const catConfig = moduleConfig?.cat || { enforce: true, block_unknown_cat: true };
+  const advConfig = moduleConfig?.adv || { enforce: true, block_unknown_adomain: true };
```

Comment:
It's probably out of scope, but I don't like these parameter names. `attr: {block_uknown_attr}` is redundant; and we do not use snake_case anywhere else. It's also inconsistent within itself (why `cat/block_unknown_cat`, but `adv/block_unknown_adomain`?) 

If we weren't following PBS this would be something like `adv: {enforce, blockUnkown}`.  Do we still prefer PBS's syntax @patmmccann ?

Reply:
i don't know, maybe those are good reasons to abandon the hope of matching the syntax.